k8ssandra-example.yaml

```diff
@@ -1,449 +0,0 @@
-apiVersion: v1
```

Comment:
Why remove this file? Granted, it has more examples for monitoring Cassandra than for creating a K8ssandraCluster resource, but it could still be useful, don't you think?

Reply:
Um, so this is embarrassing, but this was a working file that I accidentally added into the repo. I was trying to (very quietly) remove it without anyone noticing. It should never have been there because k8ssandra-operator creates the SMs and the user doesn't need this example ;)

Reply:
Heh :-) In this case, it's OK to remove.